packages/typespec-test/test/openai_generic/generated/typespec-ts/src/OpenAIClient.ts

```diff
@@ -113,209 +52,39 @@ export class OpenAIClient {
   constructor(credential: KeyCredential, options: OpenAIClientOptions = {}) {
     this._client = createOpenAI(credential, options);
     this.pipeline = this._client.pipeline;
```

Comment:
how are we going to handle pipeline ?

Reply:
lets leave this for now and will revisit in a later PR

Reply:
issue created  https://github.com/Azure/autorest.typescript/issues/2087